### ISSUE TITLE
gravity model

### ISSUE THREAD

**Issue by davcl** [jmp243/DipRep#2]
from reviewer: "a gravity model, often used in the analysis of trade relations, may provide a good baseline model, including both size of the countries and distance"

---

The Correlates of War has another dataset called the Composite Indicator of National Capability which includes "six indicators - military expenditure, military personnel, energy consumption, iron and steel production, urban population, and total population - are included in this data set.” This might be a better dataset than the Trade dataset because the span of data collection (1816 to 2012) is closer to the diplomatic exchange dataset. We can use version 4.0 since our diplomatic exchanges ends at 2005 but version 5.0 might have fixed any previous mistakes.

**Comment by davcl** [comment]
we have version 5.0 being read into the code now

**Comment by jmp243** [comment]
Yeah, so Ward 2020 uses diplomatic exchange and CINC. I think this would be an acceptable approach. so there is a nonlinear relationship with CINC and change in diplomatic ranks. Instead of focusing on conflict, I could think of a different angle like trade or cultural exchange.
